package.json

```diff
@@ -21,6 +21,7 @@
         "@types/superagent": "^4.1.16",
         "class-variance-authority": "^0.7.0",
         "clsx": "^2.1.1",
+        "cmdk": "^1.0.0",
```

Comment:
why this package over shadcn? Last update was 6 months ago. Might still be a good option though.

Also I think removing the old combobox allows us tor remove a dep.

Reply:
The way `shadcn` did it is using `Popover` and `cmdk`. There wasn't an actual package for it. Also removed `downshift` as a result of removing the old combobox